nservicebus/uow-access-to-context.include.md

```diff
@@ -0,0 +1 @@
+NOTE: In NServiceBus Versions 6 `IManageUnitsOfWorks` doesn't have access to the incoming message context. If access to headers and/or message body is needed implemement a [unit of work using behaviors instead](samples/pipeline/unit-of-work).
```

Comment:
"Versions 6 and above"  

and i think "NServiceBus " is redundant in this context?